src/Routes.js

```diff
@@ -53,23 +55,18 @@ function checkPaths(routes) {
  *      path - https://prod.foo.redhat.com:1337/insights/advisor/rules
  *      component - component to be rendered when a route has been chosen.
  */
-export const Routes = ({ childProps: { history }}) => {
+export const Routes = (props: Props) => {
```

Comment:
This shouldn't change, because we are handling redirect in `checkPaths` function.